CMakeLists.txt

```diff
@@ -532,7 +532,8 @@ endif()
 
 if(WIN32)
   # Postfix of DLLs:
-  ocv_update(OPENCV_DLLVERSION "${OPENCV_VERSION_MAJOR}${OPENCV_VERSION_MINOR}${OPENCV_VERSION_PATCH}")
+  set(BUILD_LIBS_SUFFIX "" CACHE STRING "Set a library extension to avoid conflicts in your application with other libraries.")
+  ocv_update(OPENCV_DLLVERSION "${OPENCV_VERSION_MAJOR}${OPENCV_VERSION_MINOR}${OPENCV_VERSION_PATCH}${BUILD_LIBS_SUFFIX}")
```

Comment:
I believe it is already possible to set custom `OPENCV_DLLVERSION`, so extra variable is not necessary.

Reply:
Without changing the CMake files, I dont see a way to change the created lib/dll names. Please advice an alternative to my fix.

Reply:
I was trying to say that user can set `OPENCV_DLLVERSION` to whatever they need, e.g. `500_my_custom_build` and it would work just like the new variable proposed here.